modules/database/repository.py

```
@@ -0,0 +1,111 @@
+
+    def get_database_version(self) -> str:
+        """
+        Get the version of the database
+
+        :return: The version of the database
+        """
+        try:
+            version: VersionDatabaseModel = self._database.get_version()
+            return version.semver
+        except Exception as e:
+            raise Exception(f'Error getting database version: {e}')
```

Ruff (0.8.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)